services/illuminance.md

```diff
@@ -0,0 +1,12 @@
+# Illuminance
+
+Measures light in Lux
+
+    identifier: 0x1e6ecaf2
+    extends: _sensor
+
+## Registers
+
+    ro light: u16.16 lux { typical_min=0, typical_max=6555 } @ reading
```

Comment:
I would drop the typical min (it's absolute from type)

Bright day outside is 100k lux; maybe that should be typical max? (my handheld light sensor can give readings in that range)